packages/forms-web-app/src/dynamic-forms/validator/validator.js

```diff
@@ -29,7 +29,7 @@ const validate = () => {
 				const errors = validationResult(req);
 				const mappedErrors = errors.mapped();
 
-				if (mappedErrors.length > 0) {
```

Comment:
Though it didn't cause any issues, this line wasn't actually working as far as I can see.

Reply:
makes sense, undefined is never > 0 :)